### ISSUE TITLE
what is "collective empowerment" and what does it have to do with interop?

### ISSUE THREAD

**Issue by dwsinger** [w3c/AB-public#37]
> Remain focused on interoperability and collective empowerment, and ensure our work is supported by open test suites.

?

**Comment by cwilso** [comment]
An interesting question.  This phrase (both together) has bounced around a bit, but was originally introduced in March 2021, when we adopted the strategic goals section from https://www.w3.org/wiki/AB/W3C-strategy-development#Strategic_Goals. (https://github.com/WebStandardsFuture/Vision/pull/20/commits/6686b2d63c765867eea4b373ecddd65f29aa9bdd).

That, in turn, looks like it came from @avneeshsingh on 14 Sept 2020 (https://www.w3.org/wiki/index.php?title=AB/W3C-strategy-development&action=history#:~:text=07%3A24%2C%2014%20September%202020).

My rough translation of this is that the point here is that focusing on interoperability is intimately connected with the goal of empowering the Web for everyone (rather than, say, one vendor getting to push APIs through that are non-interoperable but make the web better for them or a small set of others).  I could be mistaking the intent, and would explicitly welcome input from @avneeshsingh and @tantek.

**Comment by avneeshsingh** [comment]
Collective empowerment is explained quite well on [open standards webpage](https://open-stand.org/about-us/principles/)

**Comment by dwsinger** [comment]
I think there are (at least) 2 problems here.

1) If I ask Google to give me a definition of collective empowerment, the open standards definition doesn't come up. 
2) I think that the open standards text is not, in fact, a definition, but explores what collective empowerment might mean in the standards context.

I don't think our vision document should use terms that need external links to be comprehensible.

**Comment by cwilso** [comment]
Note that the context of this is:
"Remain focused on interoperability and collective empowerment, and ensure our work is supported by open test suites."

Although I like the concept of collective empowerment, perhaps this isn't the right place for it.  It could be dropped, or go into the vision for the web, perhaps as
"The Web is for the collective empowerment of all humanity."

**Comment by avneeshsingh** [comment]
"The Web is for the collective empowerment of all humanity." is good.
At the same time, the original use of collective empowerment in this document is for the companies which are collectively developing the open standards. Interoperability is one piece of the collective empowerment.

**Comment by cwilso** [comment]
@avneeshsingh Do you think that use of collective empowerment is important to keep in?  I think I understand the point; I'm on the fence about the term.

We could also edit the line that currently contains it to read:

"Remain focused on collectively empowering the Web by ensuring interoperability, supporting our work with open test suites."

**Comment by avneeshsingh** [comment]
"Remain focused on collectively empowering the Web by ensuring interoperability, supporting our work with open test suites." is OK.

**Comment by dwsinger** [comment]
The web is an object, and I think collective empowerment is something that applies to people, isn't it?

**Comment by cwilso** [comment]
The Web is many things, and in this usage I was already presuming it meant "the web as a platform", which *IS* collectively empowered by interop.  I mean, we could say 
"Remain focused on collectively empowering users and developers of the Web by ensuring interoperability, supporting our work with open test suites."

But that seems needlessly wordy.

**Comment by dwsinger** [comment]
I agree, needlessly wordy. Perhaps we need to state collective empowerment as its own bullet point?

In the W3C operations
* Remain focused on interoperability and ensure our work is supported by open test suites.

In our vision for the web:
* Ensure that the web principally enables collective empowerment of its users and humanity
??

**Comment by cwilso** [comment]
Coming back to this issue: I'm not excited about adding a new bullet to the "vision for the web" for collective empowerment; I'd be okay adding it to the first bullet, changing "The Web is for all humanity" to "The Web is for the collective empowerment of all humanity."  I would also be okay with not making this change.  Please thumbs-up if you think we should make this change, thumbs-down if you'd prefer to simply remove it.

In either case, I think we should remove "collective empowerment" from the point about interop and test suites.

**Comment by dwsinger** [comment]
I rather like the short punchy sentences there today. And I worry about subsetting what the web is for as it could be read as meaning only collective empowerment and not anything else (like simple enjoyment or satisfying curiosity or ...)

**Comment by TzviyaSiegman** [comment]
I don't feel strongly, but I do prefer keeping the sentence short "The web is for all humanity" is very strong as is. I don't think that "The Web is for the collective empowerment of all humanity." is as easy to understand.

**Comment by cwilso** [comment]
I think the net resolution of this should be to remove the phrase "and collective empowerment" from the document, as unnecessary.  We have a strong focus on collaboration and consensus woven throughout, this sentence works better in explaining the focus on interop without the phrase, and it seems unnecessary.

**Comment by cwilso** [comment]
@avneeshsingh I'd like your take on the above.
